packages/victory-bar/src/bar.js

```diff
@@ -77,11 +77,24 @@ const getStyle = (style = {}, props) => {
 };
 
 const evaluateProps = (props) => {
-  // Potential evaluated props are 1) `style`, 2) `barWidth` and 3) `cornerRadius`
+  /**
+   * Potential evaluated props are
+   * 1) `style`
```

Comment:
I think the original comment was a little unclear. It was meant to point out the order in which `style`, `barWidth` and `cornerRadius` are evaluated. The order for these does matter, since evaluated `style` could potentially be used when evaluating `barWidth` for example. The order for the other props doesn't matter. Can you make the distinction a little clearer in the comment?

Reply:
Ok, I will change the comment to make it more clear about the order of the props being evaluated.